### PR TITLE
Update docker pipeline

### PR DESCRIPTION
**Pull request by pranavanba**
Major changes to the ingress pipeline include:

1. Updated Dockerfile and workflow, with relevant instructions in updated README
2. Merged changes made by @itismeghasyam to the non-docker codebase version with the updated docker version

### CODE REVIEW

File: data_sync.R [hunk 1]
```diff
@@ -3,17 +3,92 @@
...
+localDirs <- list.dirs(path = AWS_DOWNLOAD_LOCATION, full.names = FALSE, recursive = FALSE) 
+
+
+#############
+# Sync Local EC2 (from ingress bucket) to pre_etl bucket (Second Sync )
+#############
+# synapser::synLogin(daemon_acc, daemon_acc_password) # login into Synapse
+synapser::synLogin()
+
+sts_token <- synapser::synGetStsStorageToken(entity = 'syn51714264', # sts enabled destination folder
```
Comment (itismeghasyam) [inline data_sync.R:56]: syn51714264 - Is the folder for ce-input-data attached to s3://recover-input-data/main

LOOKS GOOD

File: data_sync.R [hunk 2]
```diff
@@ -3,17 +3,92 @@
...
+# Sync S3 ingress bucket to Local EC2 (First Sync )
+# using prod-creds (with 'read' only permissions) for accessing the ingress bucket 
 #############
-# <<values below are test values>>
-s3SyncBuckets(source_bucket = paste0('s3://', INGRESS_BUCKET,'/'),
-              destination_bucket = paste0('s3://', PRE_ETL_BUCKET,'/'))
+# synapser::synLogin(daemon_acc, daemon_acc_password) # login into Synapse
+synapser::synLogin()
+
+sts_token <- synapser::synGetStsStorageToken(entity = 'syn52293299', # sts enabled destination folder
```
Comment (itismeghasyam) [inline data_sync.R:17]: syn52293299 - Is the folder for MHP Input Data Internal attached to 	s3://recover-ingestion/

LOOKS GOOD

File: params.R [hunk 3]
```diff
@@ -2,17 +2,20 @@
...
 
-FILE_LIST_OUTPUT = 's3files.txt' 
-# file name of the file where the aws ls output is saved
+FILE_LIST_OUTPUT = 's3files.txt'
+# # file name of the file where the aws ls output is saved
 
-SYNAPSE_PARENT_ID = 'syn51364759'
-# Synapse location where the objects are listed
\ No newline at end of file
+SYNAPSE_PARENT_ID = 'syn51714264'
```
Comment (itismeghasyam) [inline params.R:17]: Both SYNAPSE_PARENT_ID and SYNAPSE_FILEVIEW_ID both check out correctly

**Review by pranavanba — COMMENTED** [review]
All changes reviewed

**Pull request merged**